### PR TITLE
WIP: ✨ display next track if one was selected

### PR DESCRIPTION
**Pull request by hairmare**
UI parts of fixing https://github.com/radiorabe/klangbecken/issues/52.

* [x] fix text layout (esp. regarding the italic artist, kerning and uppercasing)
* [ ] do we really want this in the header
* [ ] is polling the right solution? Should we think about websockets?

## screenshot
with active next track:
Screenshot 2020-06-03 at 16 30 48
without next track in queue:
Screenshot 2020-06-03 at 16 33 01

### CODE REVIEW

**Comment by hairmare** [comment]
We need to figure out if we want to remove the `prio.m3u` playlist from the "silent" rotation or simply hide it from here if we are off air (https://github.com/radiorabe/klangbecken/issues/52#issuecomment-638304329).

**Comment by spameier** [comment]
> We need to figure out if we want to remove the `prio.m3u` playlist from the "silent" rotation or simply hide it from here if we are off air ([radiorabe/klangbecken#52 (comment)](https://github.com/radiorabe/klangbecken/issues/52#issuecomment-638304329)).

I'm in favor of keeping the button in the UI and playing the prio playlist all over again until we come on air. This would require some changes in the Liquidsoap script:
- Using `playlist` instead of `playlist.once` (already done in https://github.com/radiorabe/klangbecken/pull/58)
- Not clearing the playlist if we are off air

**Comment by hairmare** [comment]
This will be obsolete if we move forwards with https://github.com/radiorabe/klangbecken/pull/75.

**Comment by spameier** [comment]
I think we can merge this right away.

**Review by spameier — APPROVED** [review]

File: src/components/NextTrack.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import { mapGetters } from "vuex";
+
+const POLL_TIMEOUT = 3000;
+
+export default {
+  name: "NextTrack",
+  created() {
+    let poll = async () => {
+      try {
+        let response = await axios.get("/data/prio.m3u", {
```
Comment (smlz) [inline src/components/NextTrack.vue:21]: `prio.m3u` is gone. Use `"/api/player/queue/"` instead.

**Review by smlz — CHANGES_REQUESTED** [review]

File: src/components/NextTrack.vue (hunk 1 above)
Comment (hairmare) [reply src/components/NextTrack.vue:21]: I'm also gonna test if it works as intended, last I checked UX wasn't consistent with actual behavior,

Comment (hairmare) [reply src/components/NextTrack.vue:21]: (how deployed are the backend changes? 🥺)

Comment (smlz) [reply src/components/NextTrack.vue:21]: It is deployed like: it is rönning in prodökschn

Comment (smlz) [reply src/components/NextTrack.vue:21]: You get the following when `GET`-ing `/api/player/`: 
```json
{
  "uptime": "0d 19h 04m 39s",
  "liquidsoap_version": "Liquidsoap 1.3.7",
  "api_version": "0.0.15",
  "music": "c884fc50-b600-425d-bda3-b8b655fd2f03",
  "classics": "d0798bcd-e345-4742-9171-8ac78262f666",
  "jingles": "8d5ea818-2667-41b9-9b1b-42e071540bf1",
  "on_air": true,
  "current_track": {
    "id": "81922c26-43a3-4040-b60a-363269eacd02",
    "remaining": 297.4,
    "source": "music"
  },
  "queue": ""
}
```

For every playlist there is the id of the next scheduled track, plus the next scheduled track in the queue, if any.

Comment (hairmare) [reply src/components/NextTrack.vue:21]: thanks for the pointer, i gonna test if the next feature follows the behaviour i would expect given the ui while i update this branch

**Comment by hairmare** [comment]
imma close this since i would need to start from scratch anyway

**Pull request closed without merging**